### PR TITLE
Added individual transfer details view and route.

### PR DESCRIPTION
**Pull request by punalsc**
Hi @gtvj ,

Could you please review this pull request and let me know if anything needs to be changed.
If you are happy with it, please merge.

Thanks
Punal

### CODE REVIEW

File: src/routes.php [hunk 1]
```diff
@@ -12,3 +12,12 @@
 
     return $this->view->render($response, 'aggregate-transfers.html', $page_info);
 })->setName('profile');
```
Comment (gtvj) [inline src/routes.php]: Please change 'profile' to 'transfer-list' or similar

**Comment by gtvj** [comment]
Looks good - just one thing I've mentioned (changing 'profile' to 'transfer-list')

**Comment by gtvj** [comment]
Once you've made and pushed that change let me know and I'll merge the commit.

**Comment by punalsc** [comment]
Hi @gtvj ,

Done the changes you had suggested. Please review.

Thanks

**Pull request merged**